src/openlcb/SNIPClient.hxx

```diff
@@ -76,7 +76,7 @@ struct SNIPClientRequest : public CallableFlowRequestBase
 #if !defined(GTEST) || !defined(SNIP_CLIENT_TIMEOUT_NSEC)
 /// Specifies how long to wait for a SNIP request to get a response. Writable
 /// for unittesting purposes.
-static constexpr long long SNIP_CLIENT_TIMEOUT_NSEC = MSEC_TO_NSEC(1500);
+static constexpr long long SNIP_CLIENT_TIMEOUT_NSEC = MSEC_TO_NSEC(2000);
```

Comment:
I'm wondering why this is not defined as a constant instead of needing to be overridden like:
```
static long long snipTimeout = 50 * 1000000;
#define SNIP_CLIENT_TIMEOUT_NSEC snipTimeout
```
(from SNIPClient.cxxtest)

Would it make more sense to move this to a configurable constants instead? Does that work for test code?